fetch.bs

```diff
@@ -2049,6 +2051,29 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<h3 id=http-cache-partition>HTTP Cache partition</h3>
+
+<p>To determine the associated <dfn export id=http-cache>HTTP cache</dfn> partition, given
```

Comment:
I think this should define HTTP cache partition, not HTTP cache. The `id` is also redundant. So just `<dfn export>HTTP cache partition</dfn>` would work.

Reply:
Changed it to <dfn export>determining the HTTP cache partition</dfn>